api/README.md

```diff
@@ -93,7 +93,7 @@ gcloud services enable iam.googleapis.com
 
 ### Service Account
 
-DS API service(s) are a trusted application that makes authorized API calls to your GCP project service(s). The application requires a [GCP service account](https://cloud.google.com/iam/docs/service-accounts) with the appropriate permissions enabled. These permissions have been aggregated into a custom role that is associated to a service account. The custom role and associated permissions are defined in [here](./config/cds-api-mgr-role-definition.yaml)
+DS API service(s) are a trusted application that makes authorized API calls to your GCP project service(s). The application requires a [GCP service account](https://cloud.google.com/iam/docs/service-accounts) with the appropriate permissions enabled. These permissions have been aggregated into a custom role that is associated to a service account. The custom role and associated permissions are defined in [here](./config/ds-api-mgr-role-definition.yaml)
```

Comment:
Hi @mservidio, There are several other *cds* references in this file that I did not update. Can you do a complete replace `sed 's/cds/ds/g'` to clean them up. WDYT?
Thanks, Chris